metisp/pymetis/src/pymetis/recipes/instrument/metis_pupil_imaging.py

```diff
@@ -166,12 +166,30 @@ class MetisPupilImaging(MetisRecipe):
     _email = "jkarr@asiaa.sinica.edu.tw"
     _copyright = "GPL-3.0-or-later"
     _synopsis = "Basic processing of pupil images"
-    _description = (
-            "This recipe performs basic reduction (dark current subtraction, flat fielding,\n"
-            "optional bias subtraction, persistance and linearity corrections) on engineering\n"
-            "images of the pupil masks. This recipe is not expected to be used by observers\n"
-            "during regular use."
-    )
+    _description = """
+       This recipe performs basic reduction (dark current subtraction, flat fielding,\n
+       optional bias subtraction, persistance and linearity corrections) on engineering\n
```

Comment:
Perhaps check what happens with the indentation. That is, how does it look when you ask pyesorex for the description of the recipe (I don't know how to do that).

In case it prints out the indentation, then you can perhaps better remove the indentation; that is, place the text all the way to the left of the file.